imap_processing/idex/l1/idex_l1.py

```diff
@@ -1,24 +1,16 @@
-"""
-Decommutate IDEX CCSDS packets.
-
-This module contains code to decommutate IDEX packets and creates xarrays to
-support creation of L1 data products.
-"""
+"""Decommutate IDEX CCSDS packets."""
```

Comment:
This module now does something different. I would suggest saying something like what I have for CoDICE:

```
Perform CoDICE l1a processing.

This module processes decommutated CoDICE packets and creates L1a data products.
```